### PR TITLE
Start forwarding timestamps if they're present in the stream

### PR DESCRIPTION
**Pull request by Noarkhh**

### CODE REVIEW

File: test/membrane_mp3_mad/decoder_test.exs [hunk 1]
```diff
@@ -99,4 +106,18 @@ defmodule Membrane.MP3.MAD.DecoderTest do
     assert byte_size(new_state.queue) == 4
     assert new_state.native == ctx.state.native
   end
+
+  @tag :manual
+  test "manual test using Membrane.PortAudio.Sink" do
```
Comment (mat-hek) [inline test/membrane_mp3_mad/decoder_test.exs:111]: couldn't we compare with a fixture instead?

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 2]
```diff
@@ -128,10 +130,22 @@ defmodule Membrane.MP3.MAD.Decoder do
...
-      buffer_action = [buffer: {:output, %Buffer{payload: decoded_frame}}]
+      buffer_action = [buffer: {:output, %Buffer{payload: decoded_frame, pts: pts}}]
 
       <<_used::binary-size(frame_size), rest::binary>> = buffer
-      decode_buffer(native, rest, new_stream_format, buffer_action ++ stream_format_action ++ acc)
+
+      next_pts =
+        if pts == nil,
+          do: nil,
+          else: pts + RawAudio.frames_to_time(@samples_per_frame, new_stream_format)
```
Comment (mat-hek) [inline lib/membrane_mp3_mad_plugin/decoder.ex:140]: let's move it to a private function

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: test/membrane_mp3_mad/decoder_test.exs (hunk 1 above)
Comment (Noarkhh) [reply test/membrane_mp3_mad/decoder_test.exs:111]: I tried, but I was unable to generate same output as our decoder produces

Comment (mat-hek) [reply test/membrane_mp3_mad/decoder_test.exs:111]: Let's use the output from our decoder then :P

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 3]
```diff
@@ -140,19 +151,29 @@ defmodule Membrane.MP3.MAD.Decoder do
...
     end
   end
+
+  @spec get_next_timestamp(timestamp :: Membrane.Time.t() | nil, stream_format :: RawAudio.t()) ::
+          Membrane.Time.t() | nil
+  defp get_next_timestamp(timestamp, stream_format) do
+    if timestamp == nil,
+      do: nil,
+      else: timestamp + RawAudio.frames_to_time(@samples_per_frame, stream_format)
+  end
```
Comment (mat-hek) [inline lib/membrane_mp3_mad_plugin/decoder.ex:178]:
```suggestion
  defp get_next_timestamp(nil, stream_format), do: nil
  
  defp get_next_timestamp(timestamp, stream_format), do:
    timestamp + RawAudio.frames_to_time(@samples_per_frame, stream_format)
```

**Review by mat-hek — APPROVED** [review]

**Pull request merged**